tests/extension_one_api/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+if(SYCL_CTS_ENABLE_EXTENSION_ONE_API_TESTS)
```

Comment:
How about following directory structure: 
`tests/extension/<vendor>/<short_ext_name>/...`, where short_ext_name is the name w/o `ext_<vendorstring>_` [prefix](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_names_for_extensions_to_existing_classes_or_enumerations).
(E.g. `tests/extension/oneapi/sub_group_mask/...`)

Another option:
 or `tests/extension/<ext_name>/...`, where `ext_name` can be full extension name as defined by [the spec](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_names_for_extensions_to_existing_classes_or_enumerations), but w/o `ext_` prefix - it's redundant for the tests located in extension directory.
(E.g. `tests/extension/oneapi_sub_group_mask/...`)

Reply:
I think that the second option is better because we can see the vendor name in the prefix of the test group
I've applied this comment in this commit: ecc42021e507810dad1957f02176e380dfdea475